client/my-sites/sharing/buttons/buttons.jsx

```diff
@@ -41,12 +45,32 @@ class SharingButtons extends Component {
 	};
 
 	saveChanges = event => {
+		const {
+			isJetpack,
+			isLikesModuleActive,
+			siteId
+		} = this.props;
+
 		event.preventDefault();
+
 		this.props.saveSiteSettings( this.props.siteId, this.state.values );
 		if ( this.state.buttonsPendingSave ) {
 			this.props.saveSharingButtons( this.props.siteId, this.state.buttonsPendingSave );
 		}
 		this.props.recordGoogleEvent( 'Sharing', 'Clicked Save Changes Button' );
+
+		if ( ! isJetpack || isLikesModuleActive !== false ) {
```

Comment:
Is there a reason to do it this way vs `isLikesModuleActive === true` (or even just `isLikesModuleActive`)